src/main/kotlin/com/cmgapps/license/reporter/HtmlReport.kt

```diff
@@ -69,7 +69,8 @@ class HtmlReport(private val libraries: List<Library>,
                             sortBy { it.name }
                             forEach { library ->
                                 li {
-                                    +library.name
+                                    val combinedName = if (library.group.isNotEmpty()) "(${library.group}:${library.artifact})" else "(${library.artifact})"
+                                    +"${library.name} $combinedName"
```

Comment:
rather not include the `group:artifact` to the HTML output.
It is something User facing and has no context for them